app/controllers/me/index.js

```diff
@@ -5,26 +5,28 @@ export default Ember.ObjectController.extend({
     isResetting: false,
 
     actions: {
-        resetToken: function() {
+        resetToken() {
             this.set('isResetting', true);
-            var self = this;
+
             ajax({
                 dataType: "json",
                 url: '/me/reset_token',
                 method: 'put',
-            }).then(function(d) {
-                self.get('model').set('api_token', d.api_token);
-            }).catch(function(reason) {
+            }).then((d) => {
+                this.get('model').set('api_token', d.api_token);
```

Comment:
Just to clarify as well, the `this` scoping rules I guess got "fixed" for the new kinds of closures? (that'd be awesome!)